agent/go.mod

```diff
@@ -33,6 +33,7 @@ require (
 
 require (
 	cloud.google.com/go v0.94.0 // indirect
+	cloud.google.com/go/storage v1.10.0 // indirect
```

Comment:
I'm not going to pretend to understand the magic of go.mod or go.sum, but it seems really weird that these files are changing when no code of the agent is changing... do you know why that is?

Reply:
It looks like the agent module depends on the master module so that is probably why the indirect requirements are added. A new module with minimum shared code and dependencies would be needed to remove any unnecessary indirect requirements.

Reply:
got it.  Yeah, don't bother with that I think.